mantle/cmd/ore/aws/upload.go

```diff
@@ -85,6 +88,9 @@ func init() {
 	cmdUpload.Flags().StringSliceVar(&uploadGrantUsers, "grant-user", []string{}, "grant launch permission to this AWS user ID")
 	cmdUpload.Flags().StringSliceVar(&uploadGrantUsersSnapshot, "grant-user-snapshot", []string{}, "grant snapshot volume permission to this AWS user ID")
 	cmdUpload.Flags().StringSliceVar(&uploadTags, "tags", []string{}, "list of key=value tags to attach to the AMI")
+	cmdUpload.Flags().BoolVar(&uploadIMDSv2Only, "imdsv2-only", false, "enable IMDSv2-only support")
```

Comment:
If we default to true here then we could just not have any entry in `image-default.yaml` for these three knobs. i.e. right now the code has a default value and the yaml file has a default value. Maybe we just have one.

Reply:
An argument for keeping the knobs in `image-default.yaml` is that it serves as documentation for possible keys. Without it, you'd have to read the entrails of `aws.py` to figure out their names. (Of course, an alternative is documenting it elsewhere, but so far this has been a way of self-documenting the format.)

Also, not a big deal but personally I find the way `pflag` allows you to do `--boolflag=false` to turn off a flag kinda hacky. If we wanted the opposite logic, I'd prefer renaming it to e.g. `--no-imdsv2-only` instead.

Reply:
ok yeah, let's just keep it as is then.